packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1235,13 +1235,6 @@ class Transform extends SingleChildRenderObjectWidget {
   ///
   /// This is equivalent to setting an origin based on the size of the box.
   /// If it is specified at the same time as the [origin], both are applied.
```

Comment:
"at the same time" seems confusing, this all properties are applied at instantiation time.  Maybe "along with"?